docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3"
+
+services:
+  saleor-app-checkout:
+    container_name: saleor-app-checkout
+    build:
+      context: .
+      dockerfile: Dockerfile.base
```

Comment:
Is this correct?

Reply:
I'm not sure what you meant here? Both services use the same Dockerfile, they only change the command used to run the service.

`context: .` is so that they are run from the root of monorepo.

`container_name` is arbitrary tbh, we can change it